Pull in r200104 from upstream llvm trunk (by Venkatraman Govindaraju):

  removing duplicate enum value

Introduced here: http://svnweb.freebsd.org/changeset/base/262261

Index: include/llvm/Support/ELF.h
===================================================================
--- include/llvm/Support/ELF.h
+++ include/llvm/Support/ELF.h
@@ -1144,7 +1144,6 @@ enum {
   R_SPARC_REGISTER    = 53,
   R_SPARC_UA64        = 54,
   R_SPARC_UA16        = 55,
-  R_SPARC_UA16        = 55,
   R_SPARC_TLS_GD_HI22   = 56,
   R_SPARC_TLS_GD_LO10   = 57,
   R_SPARC_TLS_GD_ADD    = 58,
